src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -40,66 +40,98 @@
+    } else {
+      setUserAttributes(
+        session,
+        urlToRedirectToAfterUserLogsOut,
+        userService,
+        currentUser
+      );
+      response.sendRedirect("/user?id=" + currentUser.getKey().getId());
+    }
   }
 
-  public Entity queryForUser(UserService userService){
-      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-      Entity currentUser = null;
-      String userEmail = userService.getCurrentUser().getEmail();
-      Filter propertyFilter = new FilterPredicate(
-                "email",
-                FilterOperator.EQUAL,
-                userEmail
-        );
-       Query q = new Query("User").setFilter(propertyFilter);
-       PreparedQuery pq = datastore.prepare(q);
-       List<Entity> userEntityList = pq.asList(FetchOptions.Builder.withLimit(1));
-       if ( userEntityList.size() >= 1){currentUser = userEntityList.get(0);}
-       return currentUser;
+  public Entity queryForUser(UserService userService) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity currentUser = null;
+    String userEmail = userService.getCurrentUser().getEmail();
+    Filter propertyFilter = new FilterPredicate(
+      "email",
+      FilterOperator.EQUAL,
+      userEmail
+    );
+    Query q = new Query("User").setFilter(propertyFilter);
+    PreparedQuery pq = datastore.prepare(q);
+    List<Entity> userEntityList = pq.asList(FetchOptions.Builder.withLimit(1));
+    if (userEntityList.size() >= 1) {
+      currentUser = userEntityList.get(0);
+    }
+    return currentUser;
   }
-  
-  public void setLogInAttributes(int isLoggedIn, HttpSession session, String url, UserService userService) throws IOException {
-      String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = url;
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
 
-      session.setAttribute("isLoggedIn", isLoggedIn);
-      session.setAttribute("userEmail", userEmail);
-      session.setAttribute("logoutURL", logoutUrl);
+  public void setUserAttributes(
+    HttpSession session,
+    String url,
+    UserService userService,
+    Entity userEntity
+  )
+    throws IOException {
+    String userEmail = userService.getCurrentUser().getEmail();
+    String logoutUrl = userService.createLogoutURL(url);
+
+    session.setAttribute("isLoggedIn", 1);
+    session.setAttribute("userEmail", userEmail);
+    session.setAttribute("logoutURL", logoutUrl);
+    session.setAttribute("name", userEntity.getProperty("name"));
+    session.setAttribute("bio", userEntity.getProperty("bio"));
+    session.setAttribute("id", userEntity.getKey().getId());
   }
 
-  public void setUserAttributes(Entity userEntity, HttpSession session) throws IOException{
-      
-      session.setAttribute("username", userEntity.getProperty("name"));
+  public void setRegistartionAttributes(
```

Comment:
The same comment as before, `setRegistartionAttributes` should be `setRegistrationAttributes`.